src/pages/server-sitemap.xml/index.tsx

```diff
@@ -0,0 +1,25 @@
+import { getServerSideSitemap } from 'next-sitemap'
+import { GetServerSideProps, GetServerSidePropsContext } from 'next'
+
+export const getServerSideProps: GetServerSideProps = async (
+	ctx: GetServerSidePropsContext
+) => {
+	const response = await fetch(
+		`https://content.hashicorp.com/api/all-docs-paths`
+	)
+
+	const { result } = await response.json()
+	const fields = result.map((page: { path: string; created_at: string }) => ({
+		loc: `${__config.dev_dot.canonical_base_url}/${page.path}`,
+		lastmod: page.created_at,
+		priority: 0.7,
```

Comment:
Is there a specific reason we chose `0.7` for the value here, or is it a default?

Reply:
0.7 is default for next-sitemap, but since we are compiling an additional sitemap manually, I needed to add it in.